lisa/wa.py

```diff
@@ -645,3 +645,38 @@ def get_stats(self, **kwargs):
             agg_cols=['iteration', 'frame_id'],
             **kwargs,
         )
+
+class WASysfsExtractorCollector(WAArtifactCollectorBase):
+    NAME = 'sysfs-extractor'
+
+    def __init__(self, wa_output, path, type='diff', df_postprocess=None):
+        allowed_types = ['diff', 'before', 'after']
+
+        if type not in allowed_types:
+            self.logger.warn(
```

Comment:
Should raise `ValueError(f'{self.__class__.__qualname__} type must be one of {allowed_types}')`